UDPClientGUI/Views/ShellView.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace UDPCommGUI.Views
+{
+    /// <summary>
+    /// Interaction logic for ShellView.xaml
+    /// </summary>
+    public partial class ShellView : Window
+    {
+        public ShellView()
+        {
+            InitializeComponent();
+        }
+    }
+}
```

Comment:
Once you have MVVM, you can remove code behind files (`.xaml.cs` files)

Reply:
Deleted